### PR TITLE
[VFD-195] Structure of an annotation

### PR DESCRIPTION
**Pull request by janeek1995**

### CODE REVIEW

File: source/includes/_api_annotation.md [hunk 1]
```diff
@@ -56,7 +56,29 @@ Array Index | Attribute    | Data Type | Description
...
+d         | Display          | string              | Display parameter to choose if annotation should be displayed on the history browser timeline. Possible values are "tick", "span", or "none". Default value = "span" | False
+l         | Label            | string              | Label to be displayed above bounding box | False
+lt        | Line thickness   | string              | Preferred, not guaranteed, line thickness of bounding box. Possible values are "thin", "normal" or "thick". Default value = "thin" | False
+sc        | Box Stroke Color | string              | Preferred, not guaranteed, color as a web-safe hexadecimal value. #FFFFFF = white | False
+t         | Text             | string              | Text to be displayed as provided including newlines, line breaks, etc | False
+tt        | Tool Tip         | string              | Tooltip is to be displayed when the mouse hovers over pngSpan on timeline | False
```
Comment (Roskowski101) [inline source/includes/_api_annotation.md:79]: text width is not a string

**Review by Roskowski101 — CHANGES_REQUESTED** [review]
looks pretty good.  There are other aspects of annotations we probably need to add to this, but a good start.  Fix the one type and I think we are good

File: source/includes/_api_annotation.md (hunk 1 above)
Comment (janeek1995) [reply source/includes/_api_annotation.md:79]: updated

**Comment by mcotton** [comment]
👍

**Pull request merged**